ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +32,205 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  function  assertAddTermCommon(assert, rootNode) {
```

Comment:
Please rename to `component` since `rootNode` has the connotation of being part of the DOM